components/event-receiver/org.wso2.carbon.event.receiver.ui/src/main/resources/web/eventreceiver/eventReceiver_details.jsp

```diff
@@ -322,10 +358,13 @@
                             </td>
                         </tr>
                         <%
+                                }
                             }
                         %>
                         <%
-                            for (EventMappingPropertyDto payloadEbProperties : eventReceiverConfigurationDto.getMappingPropertyDtos()) {
+                            EventMappingPropertyDto[] payLoadMappingDto = eventReceiverConfigurationDto.getMappingPropertyDtos();
+                            if(payLoadMappingDto != null && payLoadMappingDto.length > 0){
```

Comment:
Thanks @prabathabey, Changes are done in pull request - https://github.com/wso2/carbon-analytics-common/pull/122/